index.html

```diff
@@ -1149,6 +1149,26 @@ <h3>Content Distribution Networks</h3>
       </p>
     </section>
 
+    <section class="informative">
+      <h3>Decoy Values</h3>
+
+      <p>
+The use of decoy values in status lists by <a>issuers</a> can increase the
```

Comment:
Decoy values, which are known to the issuer, also decrease the size of the group -- which is relied upon for privacy. It might be the case that random assignment of indexes in a list provides sufficient protection against whatever decoys might do, without reducing the size of the group. I'm not convinced that decoys add more value in many cases (such as for revocation status lists for which there are few revocations) than they would take away.

It might be better to say nothing about them here unless there is further analysis / text explaining the threats that they mitigate that pseudo-random assignment of indexes does not.

Reply:
Yes, I see your point.

I tried some updates in aada152a9bf66b21f529311ebd22d10e1a5d02ca and efa14d9c04f7d8216b25dcb42f1d66829071d2fa to address your concerns.

We need to say /something/ about decoys because they are being proposed as an "obvious solution" in some digital credentials initiatives even though there are known privacy harms when using them.

Any thoughts on the updated text?